Chapter-3-Exercises/CodeNeophyte_CH3_EX3_Bean_Counting.js

```diff
@@ -0,0 +1,19 @@
+function countBs (string) {
+	var numberBs = 0;
+	for (var i=0; i < string.length; i++) {
+		if (string.charAt(i) === "B") {
+			numberBs += 1;
+		}
+	}
+	return numberBs;
+}
+
+function countChar (string, B) {
```

Comment:
Looks good - one thing I'd suggest is naming the second parameter something other than `B`. It works regardless of what the name is, but if another programmer has to read/use the code, it could be important to use a name for the variable that that clearly communicates that function is intended to work for any character and not just `B`.